td-agent/Rakefile

```diff
@@ -525,6 +525,14 @@ class BuildTask
     configure_opts = [
       "--prefix=#{install_prefix}",
     ]
+    if ENV["TD_AGENT_STAGING_PATH"] and
+      (ENV["TD_AGENT_STAGING_PATH"].end_with?("el8.aarch64") or
+       ENV["TD_AGENT_STAGING_PATH"].end_with?("el7.aarch64"))
```

Comment:
Using `TD_AGENT_STAGING_PATH` to detecting the target is undesirable but I don't have better idea than it for now.

Reply:
How about using `rpm --eval %{?_target_cpu}`?

Reply:
This code is run on all platforms including Windows & debian based distros.
Detecting them is a bit bothersome than current code.